cpp/examples.cpp

```diff
@@ -150,7 +150,6 @@
 //
     auto a = new int(5);
     delete a;
-    new (&a_storage_of_callable.callable) type(forward<Callable>(a_callable));
```

Comment:
What makes this invalid?

Reply:
I don't think `new` can be used in that way in the root context.

Reply:
Ok, `example.cpp` needs quite a bit of work in that regard then. bare numbers aren't allowed in the root context as well.